Update mappings, include .tar.gz for easier use and self-document build process

gnarvaja opened the pull request:
- New version 0.0.6 generated with latest unidecode
- Change in builder.py to use repository in arbitrary path and self-document that you have to clone the python repository.
- Switch from zip to tar.gz since unzip is not available in postgres docker images, and tar / gzip are usually available (at least in Linux).
- Remove versioned sql from source, now are generated in the dist step.
- Include generated .tar.gz in the sources to be able to download it from github.

gnarvaja committed:
- Change maintainer
- user unidecode repository as external (with env variable)
- Use tar.gz instead of zip

gnarvaja committed:
Uploads .tar.gz to be able to download the built file from Git.

alexkuz commented on META.json:
```diff
@@ -1,8 +1,8 @@
-  "version": "0.0.4",
-  "maintainer": "Alexander Kuznetsov <alexkuz@gmail.com>",
+  "version": "0.0.6",
+  "maintainer": "Guillermo Narvaja <guillermo.narvaja@radiocut.fm>",
```
Can you make it a list instead? So there'd be both of us.

gnarvaja replied:
Done. Sorry I changed the maintainer before considering sending the PR, now we are both as maintainers.

gnarvaja committed:
Restore the original maintainer.

alexkuz commented:
Is the README still relevant, BTW? I didn't follow all that for a really long time.

Also, wouldn't it be better to publish .tar.gz as a release asset rather than storing it in repo?

gnarvaja commented:
Yes, perhaps adding the .tar.gz as release asset it's better. I didn't
consider that alternative. The release should be created in your repo, I
think.

About the README, I think it's fine, at least the "make && make install"
works fine, and the first section too.

--

Guillermo M. Narvaja

RadioCut - https://radiocut.fm

Phone: 54-911-6783-4435 (WhatsApp)

Telegram/WeChat: gnarvaja

Email: guillermo.narvaja@radiocut.fm

Iguazú 656 - Ciudad de Buenos Aires - Argentina



On Mon, Feb 24, 2020 at 1:02 PM ​Alexander Kuznetsov <
notifications@github.com> wrote:

> Is the README still relevant, BTW? I didn't follow all that for a really
> long time.
>
> Also, wouldn't it be better to publish .tar.gz as a release asset rather
> than storing it in repo?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/alexkuz/pg_unidecode/pull/1?email_source=notifications&email_token=AAA67473F6ZEJ56UHVPZL23REPVQ5A5CNFSM4KZBRBQKYY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGOEMYNOGA#issuecomment-590403352>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAA674YCBFUV7QIFZMD5S73REPVQ5ANCNFSM4KZBRBQA>
> .
>

gnarvaja committed:
Remove distribution file, will add it as release asset.

gnarvaja commented:
I just removed the .tar.gz from the PR and added as release asset.

--

Guillermo M. Narvaja

RadioCut - https://radiocut.fm

Phone: 54-911-6783-4435 (WhatsApp)

Telegram/WeChat: gnarvaja

Email: guillermo.narvaja@radiocut.fm

Iguazú 656 - Ciudad de Buenos Aires - Argentina



On Tue, Feb 25, 2020 at 12:00 PM Guillermo Narvaja <
guillermo.narvaja@radiocut.fm> wrote:

> Yes, perhaps adding the .tar.gz as release asset it's better. I didn't
> consider that alternative. The release should be created in your repo, I
> think.
>
> About the README, I think it's fine, at least the "make && make install"
> works fine, and the first section too.
>
> --
>
> Guillermo M. Narvaja
>
> RadioCut - https://radiocut.fm
>
> Phone: 54-911-6783-4435 (WhatsApp)
>
> Telegram/WeChat: gnarvaja
>
> Email: guillermo.narvaja@radiocut.fm
>
> Iguazú 656 - Ciudad de Buenos Aires - Argentina
>
>
>
> On Mon, Feb 24, 2020 at 1:02 PM ​Alexander Kuznetsov <
> notifications@github.com> wrote:
>
>> Is the README still relevant, BTW? I didn't follow all that for a really
>> long time.
>>
>> Also, wouldn't it be better to publish .tar.gz as a release asset rather
>> than storing it in repo?
>>
>> —
>> You are receiving this because you authored the thread.
>> Reply to this email directly, view it on GitHub
>> <https://github.com/alexkuz/pg_unidecode/pull/1?email_source=notifications&email_token=AAA67473F6ZEJ56UHVPZL23REPVQ5A5CNFSM4KZBRBQKYY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGOEMYNOGA#issuecomment-590403352>,
>> or unsubscribe
>> <https://github.com/notifications/unsubscribe-auth/AAA674YCBFUV7QIFZMD5S73REPVQ5ANCNFSM4KZBRBQA>
>> .
>>
>

alexkuz commented:
I've added you as a collaborator, feel free to create a release here
